app/Http/Validation/Entries/Upload/InputRules/RuleTimeInput.php

```
@@ -35,47 +35,56 @@ public function additionalChecks($inputDetails, $answer, ProjectDTO $project, En
-                'date' => $answer
-            ]);
+            $failedChecks[] = 'strtotime';
             $this->errors[$inputDetails['ref']] = ['ec5_79'];
         }
 
-        //Let's check if Y-m-d is actually a valid date in the history of time
+        //Let's check if H:i:s is actually a valid time
         $timePart = '';
         try {
             $timePart = explode('T', $answer ?? '')[1];
             $timePart = explode('.', $timePart ?? '')[0];
```

PHPStan (2.2.8)

[error] Variable $timePart on left side of ?? always exists and is not nullable.


(nullCoalesce.variable)